frontend/src/pages/WhatIsHealthEquity/FaqTab.tsx

```diff
@@ -107,23 +107,19 @@ Specific information on update frequencies by source can be found on our <a href
     a: `
   <ul>
     <li>
-    In an effort to be fully transparent, all data is retrieved from publicly sourced APIs and manual downloads
+      All data presented in the Health Equity Tracker are retrieved from publicly sourced application programming interfaces (APIs) and manual downloads. The data extracted from these sources are continuously updated to retrieve the most relevant data.
```

Comment:
I'd rephrase the last sentence:

"These sources are continuously updated to extract the most relevant data."